frontend/src/features/workspace/components/EmptyWorkspace/EmptyWorkspace.tsx

```diff
@@ -59,3 +60,23 @@ export const EmptyWorkspace = ({
     </Flex>
   )
 }
+
+export const EmptyDefaultWorkspace = ({
+  isLoading,
+  handleOpenCreateFormModal,
+}: EmptyWorkspacePage) => (
+  <EmptyWorkspace
+    isLoading={isLoading}
+    handleOpenCreateFormModal={handleOpenCreateFormModal}
+    title={"You don't have any forms yet"}
+    subText={'Get started by creating a new form'}
+  />
+)
+
+export const EmptyNewWorkspace = ({ isLoading }: EmptyWorkspacePage) => (
```

Comment:
/nit would prefer to have matching exported module with filename. Makes it easier to search

Reply:
[cac4a97](https://github.com/opengovsg/FormSG/pull/6754/commits/cac4a97683c5b3a7cdcaa5e96beb2d53d1991811)